src/Aspire.Dashboard/Utils/DashboardUrls.cs

```diff
@@ -45,6 +45,11 @@ public static string MetricsUrl(string? resource = null, string? meter = null, s
                 url = QueryHelpers.AddQueryString(url, "instrument", instrument);
             }
         }
+        if (dashpage is not null)
+        {
+            // dashpage must be querystring parameters because it's valid for the name to contain forward slashes.
```

Comment:
We could change this requirement. None of these exist, so we can't break any existing config.

Query parameters are currently used for single chart views, and it would be nice if we were consistent across these.

Reply:
I'm not sure what you're proposing, since this addition allows dashpages to include special query string characters, consistent with instrument